src/oidcendpoint/oidc/add_on/pkce.py

```diff
@@ -90,12 +90,13 @@ def post_token_parse(request, client_id, endpoint_context, **kwargs):
         return request
 
     try:
-        _info = endpoint_context.sdb[request["code"]]
+        _session_info = endpoint_context.session_manager.get_session_info_by_token(request["code"])
```

Comment:
Why retrieve the whole session_info if all we want is the `client_session_info`

Reply:
Sure we can add another method don't see that we gain a lot by doing it though.

Reply:
Fewer db queries, why do 3 queries to retrieve the user session, the client session and the grant session when you only want the client session